deploy/helm/distributed-compute-operator/templates/http-timeout-envoyfilter.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if and (.Values.istio.enabled) (.Values.istio.httpIdleTimeout.timeout)  }}
+{{- $justRootConfigNamespace := list .Values.istio.rootConfigNamespace }}
+{{- $defaultNamespaces := .Values.config.watchNamespaces | default $justRootConfigNamespace }}
+{{- $namespaces := .Values.istio.httpIdleTimeout.namespaces | default $defaultNamespaces }}
+{{- range $namespace := $namespaces }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
```

Comment:
qq: why create this with the helm chart and not add the resource creation to the code? just curious.

Reply:
if you were going to add the resource creation to the code instead, where and how would you do that? what are the pros and cons of that?

asking because i dont have a great answer for this

Reply:
(i did consider it but didn't find a good place in the code to do this -- i might be missing something)

Reply:
also given the nature of the issue it seemed like a good setting to have as part of the base chart

Reply:
Well, you'd have to change the resources for a given cluster type (e.g. `RayCluster`) if it's only limited to Ray. If it's required by everything, then this approach is the easier one. The only limiting aspect of this would be if you wanted to set different timeout values for say ray, dask, spark, etc... In that case, the code approach would be better IMO.

Reply:
ok. right now i think its best to apply to all the cluster types, dont really have a use case for different vals per cluster type. good to keep in mind for the future